openwillis/measures/text/util/characteristics_util.py

```diff
@@ -546,15 +376,12 @@ def pause_calculation(filter_json, measures):
     """
     for i, item in enumerate(filter_json):
         if i > 0:
-            item[measures["pause"]] = float(item["start"]) - float(
-                filter_json[i - 1]["end"]
-            )
+            item[measures["pause"]] = float(item[time_index[0]]) - float(filter_json[i - 1][time_index[0]])
```

Comment:
```suggestion
            item[measures["pause"]] = float(item[time_index[0]]) - float(filter_json[i - 1][time_index[1]])
```

Start time of the current - end time of the previous one
This is what has been causing the larger pauses

Reply:
updated